package.json

```diff
@@ -5,17 +5,53 @@
   "repository": "git@github.com:sul-cidr/histonets.git",
   "author": "",
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.25.0",
```

Comment:
Should we consider all of these "dependencies"? Some of them feel like "devDependencies"?